src/lib/util/contenthandler.js

```diff
@@ -278,6 +278,58 @@ define([
 		}
 	}
 
+	/**
+	 * Transform elements according to mapping set in configuration
+	 * Example config used for fixing Chrome 75 copy paste issues in contenteditable
+	 * Chrome will try to maintain styling on the copied range.
+	 * Depening on the styling of the copied element this may lead to <b> elements turning into <span style="font-size: 700">
+	 * Chrome is unable to copy <sub> and <sup> elements in content editable. This leads to <sub> and <sup> elements turning into spans
+	 * 
+	 *	Aloha.settings.contentHandler.handler.generic.transformFormattingsMapping: [
+	 *		{
+	 *			nodeNameIs: 'span',
+	 *			nodeNameShould: 'b',
+	 *			attribute: {
+	 *				name: 'style',
+	 *				value: 'font-weight: 700'
+	 *			}
+	 *		},
+	 *		{
+	 *			nodeNameIs: 'span',
+	 *			nodeNameShould: 'sup',
+	 *			attributes: {
```

Comment:
Should be _attribute_ instead of _attributes_.

Reply:
yes, thank you